go/runtime/host/sandbox/host.go

```diff
@@ -445,6 +445,10 @@ func (h *sandboxHost) manager(ctx context.Context) {
 
 		// Notify subscribers that the runtime has stopped.
 		h.notifier.Broadcast(&host.Event{Stopped: &host.StoppedEvent{}})
+
+		if h.cfg.Cleanup != nil {
```

Comment:
I would disallow `nil` cleanup functions and update the other constructors to set a no-op cleanup instead.

Reply:
> Why? Isn't it a nice thing that the default type still works with reasonable defaults?

Yes, this awesome. But unfortunately, you now start wondering which parameters can be `nil` and which not. So now, you need to change line 447 (2 lines above) to
```go
if notifier != nil {
    h.notifier.Broadcast(&host.Event{Stopped: &host.StoppedEvent{}})
}
```
because you are not sure. And more fields you have in the config or struct, the harder is to follow.

To avoid this, one solution is to add comment to `Cleanup` that it can be `nil`, but who reads comments? So the best solution is to program in a way that everything is not `nil`. And if that is not possible, there should be warning comments everywhere.

Reply:
Changed to make it consistent with other config options.